client/db/types.go

```diff
@@ -9,11 +9,52 @@ import (
 	"strconv"
 	"time"
 
+	"decred.org/dcrdex/dex"
 	"decred.org/dcrdex/dex/encode"
 	"decred.org/dcrdex/dex/order"
 	"github.com/decred/dcrd/dcrec/secp256k1/v2"
+	"golang.org/x/crypto/blake2s"
```

Comment:
TBH, I'm not overjoyed about importing from `golang.org/x` repos, but it's OK I guess.